### PR TITLE
Fix absl/string/numbers doesn't compile on Windows

### PR DESCRIPTION
**Pull request by MBkkt**
https://stackoverflow.com/a/22291538

### CODE REVIEW

File: absl/strings/numbers.h [hunk 1]
```diff
@@ -24,7 +24,26 @@
 #define ABSL_STRINGS_NUMBERS_H_
 
 #ifdef __SSE4_2__
```
Comment (derekmauro) [inline absl/strings/numbers.h:26]: This does not look correct to me. `__SSE4_2__` is not defined by MSVC.

How was this change tested?

**Comment by derekmauro** [comment]
I've submitted a change to our internal repo to fix this issue. It should make it to GitHub either today or Monday.

File: absl/strings/numbers.h (hunk 1 above)
Comment (MBkkt) [reply absl/strings/numbers.h:26]: `__SSE4_2__` is not defined by MSVC. Its defined by some cmake script. 
And I agree with you, its not correct.
But why msvc can't define it in future?

**Comment by MBkkt** [comment]
> I've submitted a change to our internal repo to fix this issue. It should make it to GitHub either today or Monday.

Don't see yet

**Comment by derekmauro** [comment]
It is live now.

**Comment by MBkkt** [comment]
Thanks, I will try

**Pull request closed without merging**